### PR TITLE
Remove tf1 dependency

### PR DESCRIPTION
**Pull request by krishung5**
This PR removed TF1 from the build and our QA testing.
Related PRs:
Core - https://github.com/triton-inference-server/core/pull/182
TensorFlow backend- https://github.com/triton-inference-server/tensorflow_backend/pull/90

### CODE REVIEW

**Review by dyastremsky — APPROVED** [review]

File: build.py [hunk 1]
```diff
@@ -520,29 +522,31 @@ def repoagent_cmake_args(images, components, ra, install_dir):
...
 
     cargs.append(cmake_cache_enable('TRITON_ENABLE_GPU', FLAGS.enable_gpu))
     cargs += cmake_cache_extra_args()
     cargs.append('..')
     return cargs
 
+
 def backend_repo(be):
-    if (be == 'tensorflow1') or (be == 'tensorflow2'):
+    if be == 'tensorflow2':
```
Comment (tanmayv25) [inline build.py:549]: Can we not rename tensorflow2 -> tensorflow?

File: build.py [hunk 2]
```diff
@@ -1329,8 +1328,8 @@ def create_build_dockerfiles(container_build_dir, images, backends, repoagents,
     # since we are using PyTorch, TensorFlow 1, TensorFlow 2 containers that
```
Comment (tanmayv25) [inline build.py:1328]: Should remove the mention of tensorflow 1 container.

File: build.py (hunk 1 above)
Comment (tanmayv25) [reply build.py:549]: Shouldn't this be "tensorflow" instead of "tensorflow2"?

Comment (krishung5) [reply build.py]: That was for changing "tensorflow2" to "tensorflow". Removed this check as the naming should have changed at [line 2216](https://github.com/triton-inference-server/server/pull/5589/files#diff-b70bfb027a8ef3ec4ab07a53a3b8ba83e63334ba3ce5ca6d9d773900ed452e5eR2216).

File: build.py (hunk 2 above)
Comment (krishung5) [reply build.py]: Missed that.. removed.

**Review by tanmayv25 — APPROVED** [review]

**Comment by krishung5** [comment]
@tanmayv25 I missed to update the TF lib path for L0_custom_ops. Please review, thanks!

**Review by tanmayv25 — APPROVED** [review]

**Comment by krishung5** [comment]
Thanks for the heads up. Rebased `r23.04-tf1-removed` branch in every repo including GitLab.

**Pull request merged**